tests/xbm/Makemodule.am

```diff
@@ -0,0 +1,7 @@
+if HAVE_LIBXPM
```

Comment:
guessing you mean ``HAVE_LIBX``**B**``M`` here instead of `HAVE_LIBX``**P**`M``.  either way, there is no such symbol in the autoconf code, and we always enable xbm support, so you should drop this if statement.